### DISCUSSION TITLE
Be careful what you leave in your codespaces without committing

### DISCUSSION THREAD

**Discussion by ThallyssonKlein** [community/community#36172] [Codespaces] [answered]
I come through this post to demonstrate my indignation with Github and warn other users to be careful with what they leave uncommitted in their codespaces.

A few days ago I received an email saying that a codespace of mine where I had been working for a few days without committing the changes would be deleted. I was busy, and the deadline given is ridiculously short (24h) and I lost all my work.

**Comment by thomas-sickert** [comment, marked as answer]
Thanks for the feedback. So sorry for your experience and that you lost work. 

We have a ticket to extend the notification deadline to make sure users get ample warning before the [retention](https://docs.github.com/en/codespaces/customizing-your-codespace/configuring-automatic-deletion-of-your-codespaces) period kicks in.

**Reply by ARMMWORXS** [reply]
Thanks,  Thomas.  Is #34939 my code.

**Comment by ARMMWORXS** [comment]
Security codes are registered on the Orgnization.
